crates/gitbutler-core/src/ops/reflog.rs

```diff
@@ -86,7 +86,7 @@ fn standard_signature() -> gix::actor::SignatureRef<'static> {
     gix::actor::SignatureRef {
         name: GITBUTLER_INTEGRATION_COMMIT_AUTHOR_NAME.into(),
         email: GITBUTLER_INTEGRATION_COMMIT_AUTHOR_EMAIL.into(),
-        time: Default::default(),
+        time: gix::date::Time::now_local_or_utc(),
```

Comment:
It's good that we don't care, but as a word of warning, unless the application is single-threaded or sets `time::set_soundness(Unsound)`, the timezone offset won't be available.

This is as `time` still relies on libc for this, and it isn't threadsafe. This will be an issue [until this was implemented](https://github.com/time-rs/time/issues/193).